.nocover.yaml

```diff
@@ -12,6 +12,7 @@ nocover_file_globs:
   - coala_quickstart/info_extractors/EditorconfigParsing.py
   - coala_quickstart/info_extractors/GemfileInfoExtractor.py
   - coala_quickstart/info_extractors/GruntfileInfoExtractor.py
+  - coala_quickstart/green_mode/green_mode_core.py
```

Comment:
Entire core is excluded from coverage??

Reply:
no its just the master function which calls other functions. Each individual method is tested with 100% coverage

Reply:
it needed too many mocks of methods like `os.walk()`

Reply:
80 lines ... fair enough.

Reply:
Recheck https://github.com/coala/coala-quickstart/issues/274 & https://github.com/coala/coala-quickstart/issues/273 above, and `INFO_SETTING_MAPS =` below, and maybe others below also which were your code that wasnt covered.

on a separate branch, try removing those rules and see what the coverage looks like.  Maybe it has increased in those areas to the point that a little effort will make those rules redundant.

Reply:
response at https://gitter.im/coala/coala-bears?at=5b6f97f4937eee24230624ae